extensions/windows-event-log/CollectorInitiatedSubscription.cpp

```diff
@@ -647,7 +647,10 @@ int CollectorInitiatedSubscription::processQueue(const std::shared_ptr<core::Pro
   while (renderedXMLs_.try_dequeue(xml)) {
     auto flowFile = session->create();
 
-    session->write(flowFile, &WriteCallback(xml));
+    {
+      WriteCallback wc{ xml };
```

Comment:
What's the reason behind this change?
Btw this is unused atm.

Reply:
The old version took the address of an rvalue, which is an MSVC extension, and I had compilation issues with it. It might have been because of my attempt to compile with `/std:c++latest`, but I figured fixing noncompliant code is a good thing anyway.

Reply:
The usage is on the next line

Reply:
Okay, makes sense, thanks!